src/Toolkit/I18n.php

```diff
@@ -189,12 +189,12 @@ public static function translate($key, $fallback = null, string $locale = null)
      * placeholders in the text
      *
      * @param string $key
-     * @param string $fallback
-     * @param array $replace
-     * @param string $locale
+     * @param string|array|null $fallback
+     * @param array|null $replace
+     * @param string|null $locale
      * @return string
      */
-    public static function template(string $key, $fallback = null, array $replace = null, string $locale = null)
+    public static function template(string $key, $fallback = null, ?array $replace = null, ?string $locale = null)
```

Comment:
```suggestion
    public static function template(string $key, $fallback = null, ?array $replace = null, ?string $locale = null): string
```

Also adds `string` return type 👍

Reply:
Good catch!